geospaas/nansat_ingestor/managers.py

```diff
@@ -56,6 +56,10 @@ def get_or_create(self, uri, n_points=10, uri_filter_args=None, *args, **kwargs)
         if len(uris) > 0:
             return uris[0].dataset, False
 
+        # Get name and sevice of data protocol
+        service_name = kwargs.pop('name', FILE_SERVICE_NAME)
```

Comment:
```suggestion
```
Can remove these two lines and change the method definition?

It is better to use keyword parameters explicitly rather than get them out of kwargs inside a function.

So the method definition on line 30 should be like:
```
def get_or_create(self,
        uri,
        n_points=10,
        uri_filter_args=None, 
        service_name= FILE_SERVICE_NAME,
        service= LOCAL_FILE_SERVICE,
        *args, **kwargs):
```

and then the call to the manager should also be change on line 48 in 
geospaas/nansat_ingestor/management/commands/ingest_thredds_crawl.py
```
ds_uri, _ = DatasetURI.objects.get_or_create(
    service_name=s.get('name'),
    service=s.get('service'),
    uri=s.get('url'),
    dataset=gds)
```